resolwe/flow/managers/local.py

```diff
@@ -7,12 +7,29 @@
 """
 from __future__ import absolute_import, division, print_function, unicode_literals
 
+import logging
+import subprocess
+
+from resolwe.utils import BraceMessage as __
+
 from .base import BaseManager
 
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
 
 class Manager(BaseManager):
     """Local manager for job execution."""
 
-    def run(self, data_id, script, priority='normal', run_sync=False, verbosity=1):
+    def run(self, data_id, dest_dir, argv, priority='normal', run_sync=False, verbosity=1):
         """Run process locally."""
```

Comment:
Please, document `run` method's arguments and keyword arguments in its docstring.

Maybe you can refer to `BaseManager`'s  `run` method to avoid repetition?